api_response_test.go

```diff
@@ -1497,3 +1497,42 @@ func TestGetActivitiesResponse(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateTagResponse(t *testing.T) {
+	type args struct {
+		r *APIResponse
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    Tag
+		wantErr bool
+	}{
+		{
+			name: "Create tag response",
+			args: args{
+				r: &APIResponse{
+					Values: map[string]interface{}{
+						"displayId": "123",
+					},
+				},
+			},
+			want: Tag{
+				DisplayID: "123",
+			},
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
```

Comment:
am curious to know why does this package use table tests if there is only one test case

Reply:
No specific reason. Just kept generated code as it is. (lazy me)

Reply:
This is done. 4a3f5486d9fc32e01c943c2a321c46274438bcdd